internal/pipeline/examples/simplepackage/configs/Kptfile

```diff
@@ -0,0 +1,12 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: kptfile
+pipeline:
+  mutators:
+  - image: gcr.io/kpt-fn/set-namespace
+    configMap:
+      namespace: staging
+  - image: gcr.io/kpt-fn/set-label
+    configMap:
+      tier: db
```

Comment:
@Shell32-Natsu @droot can you verify that this is the correct way to invoke the functions? I was unsure

Reply:
syntax look ok to me. I would use `tier: backend` may be so that we can use `tier: db` for the db package.

Reply:
done